packages/one-bx-shell-app/src/app/app.module.ts

```diff
@@ -16,7 +16,12 @@ import { APP_ROUTES } from './app.routes';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { AppShellModule } from '@fundamental-ngx/app-shell';
 import { HttpClientModule } from '@angular/common/http';
-import { COMPONENTS } from './components';
+import {
+  IframeLandingComponent,
```

Comment:
Remove this Landing page. and keep it only on our dashboard all together with our other plugins ., 

Also see my other comments in original PR